src/commands/load.ts

```diff
@@ -87,12 +87,7 @@ export default class Load extends FlanCommand {
         path.resolve(loadPath, input),
       ]);
     } catch (error) {
-      if (
-        flags.quiet &&
-        !/pg_restore: warning: errors ignored on restore: \d+\s*$/gim.test(
-          error.stderr
-        )
-      ) {
+      if (!flags.quiet) {
```

Comment:
should we remove the quiet flag altogether?

Reply:
Ehhh Maybe it's still worth keeping? In case there are certain errors you wanna ignore that get thrown with `--no-owner` about roles?